docs/ldml/tr35-keyboards.md

```diff
@@ -298,12 +306,85 @@ This mandatory attribute represents the locale of the keyboard using Unicode loc
   …
 </keyboard>
 ```
+
 ```xml
 <keyboard locale="fr-CH-t-k0-android">
   …
 </keyboard>
 ```
 
+_Attribute:_ `conformsTo` (required)
+
+This attribute distinguishes the keyboard from prior versions,
+and it also specifies the minimum CLDR version required.
+
+For purposes of this current draft spec, the value should always be `techpreview`
+
+```xml
+<keyboard … conformsTo="techpreview"/>
+```
+
+* * *
+
+### 5.x <a name="Element_locales" href="#Element_locales">Element: locales</a>
+
+The optional `<locales>` element allows specifying additional or alternate locales. Denotes intentional support for an extra language, not just that a keyboard incidentally supports a language’s orthography.
+
+**Syntax**
+
+```xml
+<locales>
+    <locale id="…"/>
+    <locale id="…"/>
+</locales>
+```
+
+> <small>
+>
+> Parents: `keyboard`
+> Children: `locale`
+> Occurrence: optional, single
+>
+> </small>
+
+### 5.y <a name="Element_locale" href="#Element_locale">Element: locale</a>
+
+The optional `<locales>` element allows specifying additional or alternate locales. Denotes intentional support for an extra language, not just that a keyboard incidentally supports a language’s orthography.
+
+**Syntax**
+
+```xml
+<locale id="{id}"/>
+```
+
+> <small>
+>
+> Parents: `locales`
+> Children: _none_
+> Occurrence: optional, multiple
+>
+> </small>
+
+_Attribute:_ `id` (required)
+
+> The BCP47 locale ID of an additional language supported by this keyboard.
+> Do _not_ include the `-k0-` subtag for this additional language.
+
+**Example**
+
+See [Principles for Keyboard IDs](#Principles_for_Keyboard_IDs) for discussion and further examples.
+
+```xml
+<!-- Pan Nigerian Keyboard-->
+<keyboard locale="mul-Latn-NG-t-k0-panng">
```

Comment:
I realize that _this_ example doesn't use a platform name in its locale string, in reference to my prior comment - but it's far enough after the section that I don't believe it sufficient to address that comment's concerns.  It's not in direct visual line-of-sight for comparison + contrast against the two examples seen at the part most directly defining the property.

Reply:
I think  have de emphasized platform now.